public/index.html

```diff
@@ -1,46 +1,45 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en-US">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
-      content="Web site created using create-react-app"
+      content="Smart shopping app"
     />
-    
-    <link rel="apple-touch-icon" href="logo192.png" />
-    <!--
-      manifest.json provides metadata used when your web app is installed on a
-      user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
-    -->
-    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
-    <!--
-      Notice the use of %PUBLIC_URL% in the tags above.
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
+    <meta property="og:site_name" content="TCL-19 Smart Shopping App" />
+    <meta property="og:title" content="Your smart shopping companion" />
```

Comment:
I really like the phrase here but would have a question about it (I am so not familiar with SEO). Is this sentence for the app to be found when googling? Because I do not think someone will google for **shopping companion** but **shopping app**. If not, I really like the expression, it seems "fresh". 😄